src/core/Akka/Serialization/Serialization.cs

```diff
@@ -124,19 +125,10 @@ private Serializer GetSerializerByName(string name)
             if (name == null)
                 return null;
 
-            var serializersConfig = System.Settings.Config.GetConfig("akka.actor.serializers").AsEnumerable().ToList();
-            foreach (var kvp in serializersConfig)
-            {
-                if (kvp.Key.Equals(name))
-                {
-                    var serializerTypeName = kvp.Value.GetString();
-                    var serializerType = Type.GetType(serializerTypeName);
+            if (!_serializersByName.TryGetValue(name, out Serializer serializer))
+                throw new ArgumentException($"Couldn't find serializer for [{name}] make sure you correctly defined.");
 
```

Comment:
I kept this check. Because it still possible to misconfigure for example serializer setting in persistence. Which is not checked during initialization.

Reply:
Sounds good